quicklogic/qlf_k4n8/yosys/synth.tcl

```diff
@@ -0,0 +1,32 @@
+yosys -import
+
+# Load the QuickLogic qlf_k4n8 support plugin. Note that this is only temporary
+# until support for the device is merged into the upstream Yosys
+plugin -i ql-qlf-k4n8
+yosys -import
+
+# Read VPR cells library
+read_verilog -lib $::env(TECHMAP_PATH)/cells_sim.v
+
+# Synthesize
+if {[info exists ::env(SYNTH_OPTS)]} {
```

Comment:
I think flattening this probably makes sense.  E.g.

```
set extra_opts
if {[info exists ::env(SYNTH_OPTS)]} {
  lextend extra_opts $::env(SYNTH_OPTS)
}
synth_quicklogic -family qlf_k4n8 $extra_opts
```

Reply:
I agree. Thanks for the hint. I updated the script.

Reply:
Unfortunately the approach you suggested doesn't work :/ Yosys treats `$extra_opts` as an additional argument even if the list is empty. I'd stick with the original code for now.